example/server.js

```diff
@@ -5,6 +5,6 @@ webpack.listen(8080, "localhost", function() {
   console.log("Server is listening on http://localhost:8080")
 });
 
-api.listen(7000, function() {
-  console.log('API listening at http://localhost:7000');
+api.listen(7654, function() {
```

Comment:
Yeah, I was solving a personal problem here. Typically have virtual hosts setup for projects at ports like 5000, 7000, etc.